install/kubernetes/helm/istio/charts/egressgateway/templates/deployment.yaml

```diff
@@ -12,7 +12,6 @@ metadata:
     {{ $key }}: {{ $val }}
     {{- end }}
 spec:
-  replicas: {{ .Values.replicaCount }}
```

Comment:
We still need this even with `hpa` policy, why remove this value?

Reply:
Have added default value of replica count for egressgateway deployment.